### ISSUE TITLE
Mutation @update for multiples ID

### ISSUE THREAD

**Issue by mentamarindos** [nuwave/lighthouse#1550]
**What problem does this feature proposal attempt to solve?**

Update multiple records

```
schema.graphql:
updateCheckList(ID: [ID!], status: Int): [checkList] @update

from client:
mutation{
updateCheckList(ID: [44,266,36,46], status: 1){
 ITEM
 CHECKED
}
}

```

*I've try it  but throws the error:*
```
Argument 2 passed to Nuwave\\Lighthouse\\Execution\\Arguments\\ArgPartitioner::relationMethods()
must be an instance of Illuminate\\Database\\Eloquent\\Model, instance of Illuminate\\Database\\Eloquent\\Collection given
````

**Comment by spawnia** [comment]
My initial impression is that this feature is unnecessary and quite complicated to fit in with existing functionality in Lighthouse. See https://github.com/nuwave/lighthouse/blob/master/src/Schema/Directives/UpdateDirective.php

Directive resolvers are meant to be useful abstractions for common use cases, not necessarily cover every use case ever. You can just create a resolver class through:

    php artisan lighthouse:mutation UpdateCheckList

**Comment by mentamarindos** [comment]
Yep, that's what I've been doing, I create a custom resolver but it would be nice if Lighthouse implemented it by default :)
Thanks for your response :+1:

**Comment by henzeb** [comment]
You can create your own directive that does this for you. write it once, use it twice or even thrice. Lighthouse is very flexible and extensible, just for usecases like yours. You could even make your directives into a composer package for sharing.
